File: geekster/c2/ValidTriangle.java
import java.util.Scanner;

public class ValidTriangle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("First degree- ");
        int a = sc.nextInt();
        System.out.print("Second degree- ");
        int b = sc.nextInt();
        System.out.print("Third degree- ");
        int c = sc.nextInt();
        if (a + b + c == 180) {
            System.out.println("Its a valid triangle");
        } else {
            System.out.println("Not a valid triangle");
        }
        sc.close();
    }
}

File: geekster/c2/valid_triangle.py
def isValidTriangle(a, b, c):
    if a+b+c == 180:
        return True
    return False


a, b, c = list(int(i)
               for i in input("Enter all 3 degree of the triangle to check its validity- ").split())
print(isValidTriangle(a, b, c))
